e2e/app/page/create-account-page.ts

```diff
@@ -108,6 +108,7 @@ export default class CreateAccountPage extends BasePage {
     const iframe = await findIframe(this.page, 'terms of service agreement');
     const bodyHandle = await iframe.$('body');
     await iframe.evaluate(body =>  body.scrollTo(0, body.scrollHeight), bodyHandle);
+    await this.page.waitForTimeout(1000);
```

Comment:
nit: Was this an ad-hoc fix? Most other changes are method call renames, but this looks like a behavior change.